src/runtime/components/nuxt-picture.vue

```diff
@@ -120,6 +121,7 @@ export default {
         format: this.format,
         quality: this.quality,
         background: this.background,
+        options: this.options,
```

Comment:
Same for `...this.modifiers` (and probably should be first)

Reply:
you mean ...this.options i guess and yea agreed